lumigator/python/mzai/backend/backend/tests/BUILD.pants

```diff
@@ -6,4 +6,8 @@ python_sources(
 python_tests(
     name="backend_tests",
     sources=["**/test_*.py"],
+    dependencies=[
```

Comment:
we don't need this

Reply:
Let me recheck. The tests were not finding some source code before adding these dependencies. But maybe it's not the right way to make pytest find them.

Reply:
Ok, I still have issues with these tests but they don't seem the previous errors complaining that schemas could not be resolved. AFAICT this is not present in the new SDK PRs.